src/management-system-v2/app/(dashboard)/[environmentId]/executions/[processId]/page.tsx

```diff
@@ -0,0 +1,283 @@
+      >
+        <Toolbar className={styles.Toolbar}>
+          <Space
+            aria-label="general-modeler-toolbar"
+            style={{
+              width: '100%',
+              justifyContent: 'space-between',
+              flexWrap: 'nowrap',
+              alignItems: 'start',
+            }}
+          >
+            <ToolbarGroup>
+              <Select
+                value={
+                  selectedInstanceId && selectedInstance
+                    ? selectedInstance.processInstanceId
+                    : undefined
+                }
+                onSelect={(value) => selectNewBpmn('instance', value)}
+                options={instances.map((instance) => ({
+                  value: instance.processInstanceId,
+                  label: instance.label,
+                }))}
+                placeholder="Select an instance"
+              />
+              <Tooltip title="Start new instance">
+                {/** TODO: implement start new instance */}
+                <Button icon={<PlusOutlined />} />
+              </Tooltip>
+
+              <Tooltip title="Filter by version">
+                <Dropdown
+                  menu={{
+                    items: [
+                      {
+                        key: '-1',
+                        label: selectedVersion ? (
+                          <Typography.Text disabled>none</Typography.Text>
+                        ) : (
+                          'Select a version'
+                        ),
+                        disabled: !selectedVersion,
+                      },
+
+                      ...selectedProcess.versions.map((version) => ({
+                        label: version.versionName || version.definitionName,
+                        key: `${version.version}`,
+                        disabled: false,
+                      })),
+                    ],
+                    selectable: true,
+                    onSelect: (item) => selectNewBpmn('version', +item.key),
+                    selectedKeys: selectedVersion ? [`${selectedVersion.version}`] : [],
+                  }}
+                >
+                  <Button icon={<FilterOutlined />}>
+                    {selectedVersion
+                      ? selectedVersion.versionName || selectedVersion.definitionName
+                      : undefined}
+                  </Button>
+                </Dropdown>
+              </Tooltip>
+
+              <Tooltip title="Coloring">
+                <Dropdown
+                  menu={{
+                    items: colorOptions as RemoveReadOnly<typeof colorOptions>,
+                    selectable: true,
+                    onSelect: (item) => {
+                      setSelectedColoring(item.key as ColorOptions);
+                      applyColoring(item.key as ColorOptions);
+                    },
+                    selectedKeys: [selectedColoring],
+                  }}
+                >
+                  <Button icon={<MdColorLens />} />
```

Comment:
I would maybe use "MdOutlineColorLens" and  "size={18}" to make the icon look more like the other icons in the toolbar